README.md

```diff
@@ -110,3 +110,46 @@ scoop reset deno
 ## Compatibility
 
 - The Shell installer can be used on Windows via the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/about).
+
+## Known Issues
+
+### unzip is required
+
+The program [`unzip`](https://linux.die.net/man/1/unzip) is a requirement for the Shell installer.
+
+```sh
+$ curl -fsSL https://deno.land/x/install/install.sh | sh
+Error: unzip is required to install Deno (see: https://github.com/denoland/deno_install#unzip-is-required).
+```
+
+**When does this issue occur?**
+
+During the `install.sh` process, `unzip` is used to extract the zip archive.
+
+**How can this issue be fixed?**
+
+You can install unzip via `brew install unzip` on MacOS or `apt-get install unzip -y` on Linux.
+
+### Running scripts is disabled
```

Comment:
@bartlomieju This is no longer a known issue:

```
### Running scripts is disabled
...
```

Thanks to https://github.com/denoland/deno_install/pull/133, it is fixed.

#133 removed the temporary file for installing a specific version using the PS installer. Running a temp file could lead to the following issue: `File C:\install.ps1 cannot be loaded because running scripts is disabled on this system.`

No temp file -> no "running scripts is disabled" anymore. :)